### ISSUE TITLE
opendns certificate error

### ISSUE THREAD

**Issue by pbarone** [sjmayotte/route53-dynamic-dns#18]
When using OpenDNS to check the IP address, it returns a certificate error

**Comment by zerkz** [comment]
changing env var IPCHECKER to 'ifconfig.co' worked for me

**Comment by sjmayotte** [comment]
@pbarone what OS and version?

**Comment by pbarone** [comment]
Running this in docker using the sjmayotte/route53-dynamic-dns image. Physical host is ubuntu . Tested in shell in the container with this:

/usr/src/app # wget https://diagnostic.opendns.com/myip
Connecting to diagnostic.opendns.com (34.117.80.25:443)
ssl_client: diagnostic.opendns.com: certificate verification failed: unable to get local issuer certificate
wget: error getting response: Connection reset by peer

Switching to use IPCHECKER=ifconfig.co works

**Comment by xfxian** [comment]
Can confirm I have the same issue, logs:

```
[2021-11-09T23:28:48.756] [INFO] default - HTTPS GET https://diagnostic.opendns.com/myip
[2021-11-09T23:28:48.809] [ERROR] default - Error: unable to verify the first certificate
    at TLSSocket.onConnectSecure (_tls_wrap.js:1497:34)
    at TLSSocket.emit (events.js:315:20)
    at TLSSocket.EventEmitter.emit (domain.js:467:12)
    at TLSSocket._finishInit (_tls_wrap.js:932:8)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706:12) {
  code: 'UNABLE_TO_VERIFY_LEAF_SIGNATURE'
}
```

**Comment by sjmayotte** [comment]
It appears the server for diagnostic.opendns.com is configured incorrectly.  The server's certificate chain is incomplete.

Rather than try and download the missing cert in the code I think it's better option to change the default to ifconfig.co and point opendns to ifconfig.co in the code to support backwards compatibility.

![image](https://user-images.githubusercontent.com/735527/142247242-9b082d3a-28ff-4238-894c-4a6bd492b6a4.png)

**Comment by sjmayotte** [comment]
Workaround included in v1.2.1 release that is backwards compatible.  You can pull latest from dockerhub or install from https://github.com/sjmayotte/route53-dynamic-dns/releases/latest

**Comment by ismarslomic** [comment]
Closing the issue as it seems to be stale.
